examples/test/MeasureValueFilter_test.js

```diff
@@ -164,3 +164,15 @@ test("should render data when chart is configured", async t => {
         .expect(chartValues.nth(3).textContent)
         .eql("-24,823,451");
 });
+
+fixture("Measure Value Filter Treat Null As Zero").beforeEach(
+    loginUserAndNavigate(`${config.url}/hidden/measure-value-filter-with-treat-null-as-zero`),
+);
+
+test("should filter the data with measure value filter treat null as zero", async t => {
```

Comment:
Optional: "should filter the data with measure value filter that treats measure null values as zero"